index.html

```diff
@@ -316,5 +328,127 @@ <h3 class="about-me__card__title">Skills</h3>
         </div>
       </div>
     </section>
+
+    <!-------CONTACT SECTION ------->
+    <section id="contact">
+      <div class="contact">
+        <div class="contact__container">
+          <h2>
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </h2>
+
+          <form
+            class="contact__form"
+            action="https://formspree.io/f/xknaqawg"
+            method="POST"
+          >
+            <ul>
+              <li class="contact__form__element full-name">
+                <label class="label" for="full-name">Full name:</label>
+                <input
+                  type="text"
+                  id="full-name"
+                  name="full-name"
+                  placeholder="Full name"
+                  maxlength="50"
+                  required
+                />
+              </li>
+
+              <li class="contact__form__element name">
+                <label class="label" for="firstname">First name:</label>
+                <input
+                  type="text"
+                  id="firstname"
+                  name="firstname"
+                  placeholder="First name"
+                  maxlength="30"
+                  required
+                />
+              </li>
+
+              <li class="contact__form__element lastname">
+                <label class="label" for="lastname">Last name:</label>
+                <input
+                  type="text"
+                  id="lastname"
+                  name="lastname"
+                  placeholder="Last Name"
+                  maxlength="30"
+                  required
+                />
+              </li>
+
+              <li class="contact__form__element email">
+                <label class="label" for="email">Email address:</label>
+                <input
+                  type="email"
+                  id="email"
+                  name="email"
+                  placeholder="Email address"
+                  required
+                />
+              </li>
+
+              <li class="contact__form__element message">
+                <label class="label" for="message">Your message:</label>
+                <textarea
+                  id="message"
+                  name="message"
+                  placeholder="Hey I'd like to connect to..."
+                  maxlength="500"
+                  required
+                ></textarea>
+              </li>
+
+              <li class="button__form button">
+                <button type="submit"><a href="">Get in touch</a></button>
+              </li>
```

Comment:
- [ ] As required for the project and to match the Figma design, kiindly make sure your **Button** interactions such as `hover`, `active` (when it's pressed), `disabled` and `enable` (default state) are styled as shown in the Figma template.
![Screenshot from 2023-05-02 21-49-19](https://user-images.githubusercontent.com/83042742/235770432-a8b87171-ce9c-44be-833b-c711564984f8.png)

Reply:
Again, I appreciate your check, however the project’s requirement only asked to add the hover effect. And it's added, so I kindly ask you show me where in the project asked to add the “disabled” feature, if we haven't study that topic...